### ISSUE TITLE
Implement keyboard-based manipulation with placeables

### ISSUE THREAD

**Issue by iafan** [translate/pootle#5101]
(See #3136 as a global issue on implementing shortcuts).

Let's implement the following keyboard shortcuts:

1. "Select previous placeable", mapped to `alt+shift+left`
2. "Select next placeable", mapped to `alt+shift+right`
3. "Copy placeable into the editor", mapped to `alt+shift+down`

This is how they would work:

- Once the unit is opened for editing, no placeable is highlighted. Highlighting is not remembered across unit navigation
- When one presses either `alt+shift+left` or `alt+shift+right` for the first time, the first placeable is highlighted
- Subsequent presses of these shortcuts allow one to select previous or next placeable, and they stop at first and last placeables
- If a placeable is highlighted, pressing `alt+shift+down` would replace the current selection in the textarea with the value of the placeable, keep it selected in the textarea, and also keep it highlighted in the source string.

**Comment by iafan** [comment]
This is how a currently highlighted placeable can look like:

![screenclip](https://cloud.githubusercontent.com/assets/1728158/17790813/0be93450-654d-11e6-8a1d-57ef4b175ec9.png)

This is the CSS for highlighting:

```CSS
.js-editor-copytext.highlighted
{
    background: rgba(11,134,150, 0.7);
    color: #fff;
}
```

**Comment by julen** [comment]
> * When one presses either `alt+shift+left` or `alt+shift+right` for the first time, the first placeable is highlighted
> * Subsequent presses of these shortcuts allow one to select previous or next placeable, and they stop at first and last placeables

I don't quite recall Virtaal's logic for this (and I have no way to test it on Mac, so someone else will need to confirm), but I have the feeling one could **cycle** through all elements starting from begin/end of the list of placeables. Overall the functionality was very intuitive and productive.

Note how being able to start cycling from begin/end is important: sometimes there are fuzzy translations which require adding new placeables at the end. If the only option was starting at the beginning and having to go through the entire list of placeables (provided there are more than one), this would increase the time needed to perform such operations.

**Comment by iafan** [comment]
I agree that it might be easier to cycle than to stop at boundaries. Let's do that. Also, the first time `alt+shift+left` is pressed (and there's no highlighted placeable) we can highlight the _last_ one. This way you can easily start going backwards from the end.
